flowetl/etl/etl/production_task_callables.py

```diff
@@ -8,7 +8,60 @@
 """
 import logging
 
-from airflow.models import DagRun
+from pathlib import Path
+
+from airflow.models import DagRun, BaseOperator
+from airflow.hooks.dbapi_hook import DbApiHook
+
+
+def render_sql_callable(
+    *,
+    dag_run: DagRun,
+    task: BaseOperator,
+    db_hook: DbApiHook,
+    config_path: Path,
+    template_name: str,
+    **kwargs,
+):
+    """
+    This function takes information from the DagRun conf to locate
+    the correct sql template file, uses the DagRun conf to populate the
+    template and runs it against the DB.
+
+    Parameters
+    ----------
+    dag_run : DagRun
+        Passed as part of the Dag context - contains the config.
+    task : BaseOperator
+        Passed as part of the Dag context - access to the instantiated
```

Comment:
```suggestion
        Passed as part of the Dag context - provides access to the instantiated
```
(Is this what you mean here?)